content/_includes/utility.macros.njk

```diff
@@ -32,7 +32,7 @@ params:
   attr,
   value=none
 ) %}
-  {%- if (value and (value | string | lower == 'true')) or (value == '') -%}
+  {%- if (value and (value | string | lower == 'true')) or (value === '') -%}
```

Comment:
@mirisuzanne This was a longstanding bug IMO, causing `value=false` to render `attr` anyway. For example, inputs with `required=false` would render with the `required` attr.